.travis.yml

```diff
@@ -1,34 +1,70 @@
-#This file is generated by ModuleSync, do not edit.
 ---
 sudo: false
+dist: trusty
 language: ruby
 cache: bundler
-script: "bundle exec rake release_checks"
+before_install:
+  - bundle -v
+  - rm Gemfile.lock || true
```

Comment:
I think ```rm -f``` is cleaner than ```|| true```

Reply:
Made a PR for this against the template https://github.com/puppetlabs/pdk-templates/pull/35